crates/matrix-sdk-ui/src/timeline/controller/state_transaction.rs

```diff
@@ -295,6 +295,9 @@ impl<'a> TimelineStateTransaction<'a> {
                                 }
                             }
                         }
+                        TimelineFocusKind::Thread => {
+                            should_add = true;
```

Comment:
(This is going to change in the PR I've made, which I'll land after yours, but: no need to do anything in this match arm, since `should_add` is true (line 264))

Reply:
Uh, good point. I rebased this on top of your PR and got rid of it.